src/carousel.tsx

```diff
@@ -680,13 +680,18 @@ type SlideshowProps = {
  * This context is used internally to let other components (Hero) now if they are rendered inside a Slideshow
  * to make some tweaks in the UI
  */
-const IsInsideSlideshowContext = React.createContext(false);
+const SlideshowContext = React.createContext<{withBullets: boolean} | undefined>(undefined);
 
-export const useIsInsideSlideshowContext = (): boolean => React.useContext(IsInsideSlideshowContext);
+export const useSlideshowContext = (): {withBullets: boolean} | undefined =>
+    React.useContext(SlideshowContext);
 
-export const IsInsideSlideshowProvider = ({children}: {children: React.ReactNode}): JSX.Element => (
-    <IsInsideSlideshowContext.Provider value>{children}</IsInsideSlideshowContext.Provider>
-);
+export const SlideshowProvider = ({
```

Comment:
why is this exported?

also, I think you don't need this component, just use `SlideshowContext.Provider`

Reply:
Not sure why it was being exported. I've removed it